draft-ietf-quic-http.md

```diff
@@ -379,11 +388,14 @@ An HTTP message (request or response) consists of:
 3. optionally, trailing headers, if present (see {{!RFC7230}}, Section 4.1.2),
    sent as a single HEADERS frame.
 
+Receipt of DATA and HEADERS frames in any other sequence MUST be treated as a
+connection error of type H3_FRAME_UNEXPECTED ({{errors}}).
+
 A server MAY send one or more PUSH_PROMISE (see {{frame-push-promise}}) or
 DUPLICATE_PUSH (see {{frame-duplicate-push}}) frames before, after, or
 interleaved with the frames of a response message. These PUSH_PROMISE and
 DUPLICATE_PUSH frames are not part of the response; see {{server-push}} for
-more details.
+more details.  These frames are not permitted in pushed responses.
```

Comment:
```suggestion
more details.  These frames MUST NOT be sent in pushed responses.
```
or

```suggestion
more details.  A pushed response is malformed if it includes PUSH_PROMISE or DUPLICATE_PUSH frames.
```

Reply:
Fixed, but I'm going to reverse that.  I like Kazuho's logic that misuse of a frame is a connection error, because the peer is the only one who could have generated it.  I'm going to make these H3_FRAME_UNEXPECTED instead.